js/battle.js

```diff
@@ -524,18 +524,21 @@ var Pokemon = (function () {
 			this.baseAbility = ability;
 		}
 	};
+	Pokemon.prototype.prepName = function (name) {
```

Comment:
I'm not sure what you're asking?

I'm thinking you should have a `htmlName` function that replaces `prepName` entirely. `getFullName` wouldn't call `prepName` but would need to reimplement wrapping the name in a span.